andela/activo-web
# #160077335 Fix additional field not repositioned with the values

**What does this PR do?**

This PR  fixes the bug for repositioning attribute values when been dragged.

**Description of Task to be completed?**

On the edit category page or new category page, when a user clicks on `add  new attribute`, selects `dropdown` for the input type and adds options to it.  If the additional field is dragged to a new position, it retains the options for that attribute.


**How should this be manually tested?**

Navigate to `Settings > Categories > New|Edit Category`
Go to the additional field section and add two attributes with different values/choices
Drag any of the additional fields to reposition it
Check the choices/values underneath the additional attribute in its new position.

**Any background context you want to provide?**

N/A

**What are the relevant pivotal tracker stories?**

[#160077335](https://www.pivotaltracker.com/story/show/160077335)

**Screenshots (if appropriate)**

<img width="1440" alt="screen shot 2018-08-30 at 8 04 41 pm" src="https://user-images.githubusercontent.com/14803275/44874648-c29c6700-ac93-11e8-9697-ff4a61d080a3.png">

<img width="1440" alt="screen shot 2018-08-30 at 8 04 54 pm" src="https://user-images.githubusercontent.com/14803275/44874665-d34cdd00-ac93-11e8-8b05-b0c75c4b62c6.png">


**Questions:**

N/A
